articulation_msgs/ModelMsg model	# single kinematic model
std_msgs/Duration target_duration	# duration of movement
---
std_msgs/Bool success
std_msgs/String error_message
